api/src/db/models/user_models.py

```diff
@@ -60,3 +61,14 @@ class LoginGovState(ApiSchemaTable, TimestampMixin):
 
     # https://openid.net/specs/openid-connect-core-1_0.html#NonceNotes
     nonce: Mapped[uuid.UUID]
+
+
+class UserSavedOpportunity(ApiSchemaTable, TimestampMixin):
+    __tablename__ = "user_saved_opportunity"
+
+    user_id: Mapped[uuid.UUID] = mapped_column(ForeignKey(User.user_id), primary_key=True)
+    opportunity_id: Mapped[int] = mapped_column(
+        ForeignKey(Opportunity.opportunity_id), primary_key=True
+    )
```

Comment:
Oh, right we'd want this to be a BigInteger so that the type matches opportunity ID exactly

Reply:
Updated this!